cfpb/sbl-frontend
# fix: [Update financial profile] Update buttons

Close #607 

## Changes

- Remove `Go back` button
- Replace `Save and continue` button with a `Submit` button
- Update secondary button label to `Reset form`

## How to test this PR

1. Landing -> Institution profile -> Update institution profile
2. `Submit` should be disabled by default
3. Upon changing form data, `Submit` becomes enabled
4. Resetting form disabled the `Submit` button
5. Successfully submitting the form redirect to a `/summary` page

## Screenshots

|State|Shot|
|---|---|
|Default|![Screenshot 2024-05-29 at 3 53 37 PM](https://github.com/cfpb/sbl-frontend/assets/2592907/78cce31c-a1e1-4714-96c5-683e8af05feb)|
|Form data changed, Reset active|![Screenshot 2024-05-29 at 3 53 31 PM](https://github.com/cfpb/sbl-frontend/assets/2592907/f3fef1c5-7bd0-41b5-9970-3dc781533475)|
|Submit active and highlighted|![Screenshot 2024-05-29 at 3 53 26 PM](https://github.com/cfpb/sbl-frontend/assets/2592907/8b2070f7-2836-4f12-8f55-d2acfe568ae1)|

Full form, default 
![screencapture-localhost-8899-institution-123456789TESTBANK456-update-2024-05-29-16_01_17](https://github.com/cfpb/sbl-frontend/assets/2592907/e55a2d48-6598-41ca-8069-ba63746d7b99)
